encoding.h

```diff
@@ -203,6 +203,7 @@
 #define IRQ_HOST      13
 #define IRQ_S_CLUSTER 17
 #define IRQ_M_CLUSTER 19
+#define IRQ_M_EXT     20
```

Comment:
This overrides the previous definition of `IRQ_M_EXT`, see line 200.

Indeed, external interrupts are already a thing, defined by the RISC-V specification. I would suggest to call this _accelerator_ interrupt, and use the `ACC` abbreviation.